src/game/APISocket.h

```diff
@@ -229,10 +229,8 @@ class CAPISocket {
     std::queue<DataPack *> m_qRecvPkt;
 
 #ifdef _DEBUG
-    __SocketStatisics m_Statistics_Send_Sum[255];
-    __SocketStatisics m_Statistics_Recv_Sum[255];
-//    std::vector<__SocketStatisics> m_Statistics_Send[255];
-//    std::vector<__SocketStatisics> m_Statistics_Recv[255];
+    __SocketStatisics m_Statistics_Send_Sum[UCHAR_MAX + 1];
+    __SocketStatisics m_Statistics_Recv_Sum[UCHAR_MAX + 1];
```

Comment:
Nice! Good catch and thanks for the PR.
Small nitpick, since we removed `m_Statistics_Send` and `m_Statistics_Recv` from the comments there, just to not raise future questions for other references when looking into the code, I also found this that we can remove: https://github.com/ko4life-net/ko/blob/3badcad01edf2e59775d697e474864a12159ddda/src/game/APISocket.cpp#L419-L422

Reply:
What does this do? Can you explain it to me in average Joe language?

Reply:
I removed it now. I also updated the other comments.

Reply:
> What does this do? Can you explain it to me in average Joe language?

Nothing because it's commented out. But it evidently used to store its packet debug in a dynamic vector, instead of in an array.